framework/db/mysql/QueryBuilder.php

```diff
@@ -43,6 +43,7 @@ class QueryBuilder extends \yii\db\QueryBuilder
         Schema::TYPE_BINARY => 'blob',
         Schema::TYPE_BOOLEAN => 'tinyint(1)',
         Schema::TYPE_MONEY => 'decimal(19,4)',
+        Schema::TYPE_JSON => 'json'
```

Comment:
It's better to use `Schema::TYPE_JSON => 'jsonb'`
https://www.postgresql.org/docs/current/static/datatype-json.html